NAMESPACE

```diff
@@ -1,7 +1,9 @@
 # Generated by roxygen2: do not edit by hand
 
+export("%>%")
```

Comment:
i'd rather not import magrittr since it's not used internally, users can easily load it if needed, and likely will be loaded anyway since I bet most will have dplyr/tidyverse loaded

Reply:
Ok no problem -- I think this was from `devtools::use_pipe()` when `multi_color` was in the package and used the pipe. Now it's not needed, like you said